### ISSUE TITLE
Gettting Started Not working with latest quarkus plugin version 1.1.0.CR1

### ISSUE THREAD

**Issue by Anand0583** [quarkusio/quarkus-quickstarts#401]
When the getting-started is upgraded to 1.1.0.CR1 version then build is failing with the following error.

`[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.1.0.CR1:build (default) on project getting-started: Failed to build a runnable JAR: Failed to augment application classes: Property quarkus.http.auth.form.landing-page not found `

pom.xml
` <quarkus-plugin.version>1.1.0.CR1</quarkus-plugin.version>
    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
    <quarkus.platform.version>1.0.1.Final</quarkus.platform.version>
`

Getting Started with 1.0.1.Final is failing because of the below issue.

https://github.com/quarkusio/quarkus/issues/5335

Thanks.

**Comment by northbyte** [comment]
I've encountered the same when upgrading my application from `1.0.1.Final` to `1.1.0.CR1` as well.
Unfortunately I cannot share the code, however just by compiling on jdk8 seems to make it fail.

```
> java -version
openjdk version "1.8.0_232"
OpenJDK Runtime Environment (build 1.8.0_232-b09)
OpenJDK 64-Bit Server VM (build 25.232-b09, mixed mode)
```

**Comment by agoncal** [comment]
I have the same problem with 1.1.1.Final

**Comment by cescoffier** [comment]
Can you check that the Qurkus version and the platform versions are aligned?

**Comment by agoncal** [comment]
Yes, you are right. It works with:

```
    <quarkus-plugin.version>1.1.1.Final</quarkus-plugin.version>
    <quarkus.platform.version>1.1.1.Final</quarkus.platform.version>
```

Slightly confusing to have two different versions. I suppose if they evolve separatelly it make sense, but if they both follow the same pace and release, it could help to only have one version.

**Comment by cescoffier** [comment]
@maxandersen I let you comment on this.

**Comment by maxandersen** [comment]
I think this another example of where tools break due to core change or vice versa; @aloubyansky are you aware of this one ?

**Comment by aloubyansky** [comment]
No, I wasn't. For people wanting update: changing `quarkus-plugin.version` does not affect the Quarkus core and other extension versions, it's only the maven plugin version (which may and probably will have its own release cycle in perspective).
To actually upgrade to the latest (or switch in general to another) version of Quarkus and its extensions you should be changing `quarkus.platform.version`.
You may even try leaving `quarkus-plugin.version` unchanged.
